src/System.Private.Uri/src/System/UriHelper.cs

```diff
@@ -742,16 +789,19 @@ internal static bool IsLWS(char ch)
         }
 
         //Only consider ASCII characters
-        internal static bool IsAsciiLetter(char character)
-        {
-            return (character >= 'a' && character <= 'z') ||
-                   (character >= 'A' && character <= 'Z');
-        }
-
-        internal static bool IsAsciiLetterOrDigit(char character)
-        {
-            return IsAsciiLetter(character) || (character >= '0' && character <= '9');
-        }
+        internal static bool IsAsciiLetter(char character) =>
```

Comment:
I don't know if this is in a hot path, but:
```cs
internal static bool IsAsciiLetter(char character) => (((uint)character - 'A') & ~0x20) < 26;
```

Reply:
Thanks.